src/components/BookInfo/BookInfo.tsx

```diff
@@ -8,13 +8,16 @@ import React, {
 import { useParams } from "react-router-dom";
 import axios from "axios";
 import { fetchUrl, renderAuthors } from "../../helpers/BookHelper";
-import { notification, Skeleton } from "antd";
+import { Badge, notification, Skeleton, Typography } from "antd";
 // @ts-ignore
 import StarRatings from "react-star-ratings";
```

Comment:
No types for this :C ?

Reply:
no :(
bas sam piknul komponentu koja nema